ariadne_codegen/graphql_schema_generators/utils.py

```diff
@@ -13,11 +13,19 @@
 )
 
 
+def _keep_dict_tuple_capitalized(obj):
+    if isinstance(obj, dict):
+        return "Dict"
+    if isinstance(obj, tuple):
+        return "Tuple"
+    return obj.__class__.__name__
+
+
 def get_named_type(type_: GraphQLNamedType, type_map_name: str) -> ast.Call:
     return generate_call(
         func=generate_name("cast"),
         args=[
-            generate_name(type_.__class__.__name__),
+            generate_name(_keep_dict_tuple_capitalized(type_)),
```

Comment:
Hmm, I don't see a point in this change. `type_` has the annotation `GraphQLNamedType`, so it shouldn't get `dict` or `tuple`. Am I missing something? Do you see a case where it's necessary?